config/brakeman.ignore

```diff
@@ -207,6 +207,26 @@
       "user_input": null,
       "confidence": "Medium",
       "note": "Explicitly configured years ago in https://github.com/department-of-veterans-affairs/vets-api/commit/c73c8fc5cc23262e8f708fad0f7c1052f9c88a7b"
+    },
```

Comment:
Can you make a note in the PR description as to why this is a brakeman exception

Reply:
Modified the PR description to address the comment